android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -65,9 +66,10 @@
   private static final RegionResolver REGION_RESOLVER = new RegionResolver();
   private static final String SEPARATOR = "\0";
 
-  public static Comparator<PwPair> newDistanceComparator() {
+  public static Comparator<PwPair> newDistanceComparator(final Set<String> fav) {
```

Comment:
"DistanceComparator" is no longer an appropriate name for this comparator, could you pick a new name and add a comment describing its behavior?

Also, please rename the fav parameter to "favoriteUrls"